flycheck.el

```diff
@@ -7884,6 +7885,21 @@ See URL `http://jruby.org/'."
   :modes (enh-ruby-mode ruby-mode)
   :next-checkers ((warning . ruby-rubylint)))
 
+(flycheck-def-config-file-var flycheck-reekrc ruby-reek "config.reek"
+  :safe #'stringp)
+
+(flycheck-define-checker ruby-reek
+  "A Ruby smeel checker using reek
+
+See URL `https://github.com/troessner/reek'."
+  :command ("reek" "--format=xml"
+            (config-file "--config" flycheck-reekrc)
+            source-original)
```

Comment:
Because I didn't read all the documentation to know how to do a checker. Sorry. I simplified the definition I hope this is better.